osutil/rename.go

```diff
@@ -19,6 +19,8 @@
 
 package osutil
 
+// SwapDirs swaps oldpath with newpath in an atomic way from the point
+// of view of the running system.
```

Comment:
Maybe this should also say that it is up to the caller to call sync when appropriate?